embedded_postgres_test.go

```diff
@@ -231,7 +231,7 @@ func Test_ErrorWhenCannotStartPostgresProcess(t *testing.T) {
 
 	err = database.Start()
 
-	assert.EqualError(t, err, fmt.Sprintf(`could not start postgres using %s/bin/pg_ctl start -w -D %s/data -o "-p 5432"`, extractPath, extractPath))
+	assert.EqualError(t, err, fmt.Sprintf(`could not start postgres using %s/bin/pg_ctl start -w -D %s/data -o -p 5432`, extractPath, extractPath))
```

Comment:
exec.Command actually ensures each of the arguments are passed directly, so we don't need to wrap the whole options string in quotes - in fact, if you do (as I did initially) it causes problems. See https://stackoverflow.com/a/26473771